### ISSUE TITLE
Missing gtk2 theme in Yaru

### ISSUE THREAD

**Issue by eaglersdeveloper** [ubuntu/yaru#654]
When I opened Inkscape, I saw an interface in Win95 style:
![2018-07-26 15-32-32](https://user-images.githubusercontent.com/40228953/43260324-336bdbb6-90ea-11e8-8117-31ed3fa18d26.png)

Yaru theme was setted:
![2018-07-26 15-33-51](https://user-images.githubusercontent.com/40228953/43260725-5aed8602-90eb-11e8-8222-1f8fa16ca774.png)

I change theme to Communitheme:
![2018-07-26 15-34-27](https://user-images.githubusercontent.com/40228953/43260787-7cb004ea-90eb-11e8-8cbf-0db1ba05ac99.png)

**Comment by Feichtmeier** [comment]
I think that had been fixed by merljin . Could you try again?

**Comment by clobrano** [comment]
Tried now, confirm the issue with inkscape, but it also prints this error but without using snap

**Comment by clobrano** [comment]
It seems the same with adwaita and in ubuntu session though, so I guess it's a inkscape problem (gimp works fine for example)

**Comment by Pitel** [comment]
I have the same problem in 18.10. :(

Not using snap, also happens in Deluge torrent client.

**Comment by clobrano** [comment]
@Pitel, I can not reproduce this issue on 18.10. Have you upgraded to 18.10 from a preinstalled 18.04 or you did a clean install?
Could you also copy here the result of these 2 commands?
    
    $ gsettings get org.gnome.desktop.interface gtk-theme
    $ ls -l /usr/share/themes/Yaru/gtk-*

Thanks

**Comment by Pitel** [comment]
Upgraded

```
'Yaru-dark'
```

```
drwxr-xr-x 2 root root  12288 říj 18 20:28 assets
-rw-r--r-- 1 root root 309774 říj 10 10:05 gtk.css
-rw-r--r-- 1 root root 309054 říj 10 10:05 gtk-dark.css
```

**Comment by Pitel** [comment]
Oh, I just tried normal Yaru (not dark) and it looks ok. :+1:

**Comment by clobrano** [comment]
Thanks @Pitel, Yaru-dark is currently missing the GTK2 part, this is the reason of this issue

**Comment by clobrano** [comment]
One more question @Pitel, have you found Yaru-dark only after the upgrade, or did you tried it somehow in Bionic?

**Comment by dhd0004** [comment]
I am still having the same problem with yaru-dark

**Comment by clobrano** [comment]
@dhd0004 on which system are you? Ubuntu 18.04 (and so is the snap channel stable or edge?) or in 18.10 already?

**Comment by dhd0004** [comment]
Ubuntu 18.10, clean installation.

**Comment by clobrano** [comment]
18.10 does not have the fix yet, only 18.04 snap installation of edge channel has it now

**Comment by minghuaw** [comment]
So I guess a temporary solution could be copying Yaru's gtk-2.0 folder to Yaru-dark?
